packages/mini-cart/src/mini-cart.tsx

```diff
@@ -0,0 +1,105 @@
+import { CheckoutProvider, Button } from '@automattic/composite-checkout';
+import { useShoppingCart } from '@automattic/shopping-cart';
+import styled from '@emotion/styled';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import { MiniCartLineItems } from './mini-cart-line-items';
+import type { ResponseCart } from '@automattic/shopping-cart';
+
+const MiniCartWrapper = styled.div`
+	box-sizing: border-box;
+	padding: 16px;
+	max-width: 480px;
+	text-align: left;
+	font-size: 1rem;
+`;
+
+const MiniCartHeader = styled.div`
+	text-align: left;
+`;
+
+const MiniCartTitle = styled.h2`
+	font-weight: 600;
+`;
+
+const MiniCartSiteTitle = styled.span`
+	color: var( --color-neutral-50 );
+	font-size: 0.875rem;
+`;
+
+const MiniCartFooter = styled.div`
+	margin-top: 12px;
+`;
+
+const MiniCartTotalWrapper = styled.div`
+	font-weight: 600;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	margin-bottom: 4px;
+`;
+
+function MiniCartTotal( { responseCart }: { responseCart: ResponseCart } ) {
+	const { __ } = useI18n();
+	return (
+		<MiniCartTotalWrapper className="mini-cart__total">
+			<span>{ __( 'Total' ) }</span>
+			<span>{ responseCart.total_cost_display }</span>
+		</MiniCartTotalWrapper>
+	);
+}
+
+export function MiniCart( {
+	selectedSiteSlug,
+	goToCheckout,
+}: {
+	selectedSiteSlug: string;
+	goToCheckout: ( siteSlug: string ) => void;
+} ): JSX.Element {
+	const {
+		responseCart,
+		removeCoupon,
+		removeProductFromCart,
+		isLoading,
+		isPendingUpdate,
+	} = useShoppingCart( selectedSiteSlug );
+	const { __ } = useI18n();
+	const isDisabled = isLoading || isPendingUpdate;
+	const isPwpoUser = false; // TODO: deal with this properly
+
+	return (
+		<CheckoutProvider paymentMethods={ [] } paymentProcessors={ {} }>
```

Comment:
Do I get it right that we're consciously avoiding providing payment methods here? Is that because we won't really be checking out in this interface and that allows us to keep things simple and clean? 

I wonder if need everything that `CheckoutProvider` provides, or we can just go along with some of its internals, like `≤LineItemsProvider />` for example.

Reply:
That's a good idea, although at the current moment the sub-providers are undocumented parts of the API and I suspect some things might not work correctly without the React context provided by `CheckoutProvider`.

My inclination is to leave this with empty arguments for the time being and iterate on promoting simpler providers in a later PR. Nonetheless, it may be worth adding a comment here to explain what's going on and perhaps making sure the empty arguments use unchanging objects.

Reply:
I agree that adding a comment there to make the intentions transparent will suffice for now 👍